src/v2/view-builder/views/phone/ChallengeAuthenticatorDataPhoneView.js

```diff
@@ -31,12 +31,14 @@ const Body = BaseForm.extend(
       const sendText = ( this.model.get('primaryMode') === 'sms' )
         ? loc('oie.phone.verify.sms.sendText', 'login')
         : loc('oie.phone.verify.call.sendText', 'login');
+      const carrierChargesText = loc('oie.phone.carrier.charges', 'login');
       const extraCssClasses =
         this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login') ?
           'strong no-translate' : '';
       // Courage doesn't support HTML, hence creating a subtitle here.
       this.add(`<div class="okta-form-subtitle" data-se="o-form-explain">${sendText}
         <span ${ extraCssClasses ? 'class="' + extraCssClasses + '"' : ''}>${this.model.escape('phoneNumber')}</span>
+        <div style="margin-top: 15px;">${carrierChargesText}</div>
```

Comment:
We don't add any css directly into `.js`.. Please add css to `/assets/sass/v2/` 

Also it would be better if you can use something existing.. Try `<p>` tag.

Reply:
file `_phone-challenge.scss` to be precise..

```
  .okta-form-subtitle {
    text-align: center;
    
    p {
    margin-top : 1em;
    }
  }
 ```

Reply:
Ack. Moved it to `_phone-challenge.scss`